app/styles/play/level/tome/spell.sass

```diff
@@ -28,18 +28,27 @@
     border: 1px solid transparent
 
   span.code-background
-    border-width: 124px 76px 64px 40px
+    border-width: 115px 76px 64px 40px
     border-image: url(/images/level/code_editor_background_border.png) 124 76 64 40 fill stretch
 
   img.code-background
     display: none
     width: 100%
 
-  .save-status
-    display: none
+  .programming-language-container
     position: absolute
-    top: 1%
-    right: 4%
+    top: 10px
+    right: 26px
+    z-index: 4
+    font-family: "Open Sans Condensed", "Helvetica Neue", Helvetica, Arial, sans-serif
+    font-size: 14px
+    padding: 0px 10px
+    .programming-language-label
+      color: #6E654B
+      font-weight: bold
+      text-transform: uppercase
+    .programming-language
+      color: #1300BE
```

Comment:
Good point.  Styling from @ryang217, so I'll sync up with her about making it less linkish.